internal/pkg/grpcutil/logger.go

```diff
@@ -131,15 +134,52 @@ func setRealIPAddress(ctx context.Context) {
 				continue
 			}
 
-			addr, err := netip.ParseAddr(peer[0])
-			if err != nil {
-				continue
+			for str := range splitString(peer[0], ",") {
```

Comment:
Why can't we just use this?
```suggestion
			for str := strings.Split(peer[0], ",") {
```

Reply:
Just an optimization. String split returns a slice, which we don't need. We can delete this code, once Go 1.24 lands https://go-review.googlesource.com/c/go/+/587095

Reply:
lets put a TODO comment here then

Reply:
Okay, will do that.